modules/deploy/pages/deployment-option/cloud/serverless.adoc

```diff
@@ -0,0 +1,154 @@
+= Provision a Serverless Cluster
+:description: Learn how to create a Serverless Cloud cluster.
+:page-cloud: true
+:page-beta: true
+
+Redpanda Serverless is the easiest way to test event streaming in the cloud: You can start streaming 30 seconds after clicking to create the cluster. 
+
+[NOTE]
+====
+To use our Beta release, Redpanda issues you enough credits to run Redpanda with reasonable throughput. These credits expire 14 days after you activate your tenant through email verification. For details, see <<Beta credits and limits>>. To ask questions, give feedback, or extend your trial, use the **#serverless** https://redpandacommunity.slack.com/[Community Slack^] channel, or contact https://support.redpanda.com/hc/en-us/requests/new[support^].
+==== 
+
+== Prerequisites
+
+. https://redpanda.com/try-redpanda?section=cloud-trial[Sign up for a Beta trial^] of Redpanda Serverless.
+
+. Make sure you have the latest version of `rpk`. See xref:get-started:rpk-install.adoc[].
+
+== Create a Serverless cluster
+
+For the Beta release, you'll receive an email to activate your Serverless trial. 
+
+. In your Welcome email, click to activate your trial. 
++
+Redpanda creates an organization and namespace for your multi-tenant Serverless cluster in AWS.
+
+. Log in to Redpanda Cloud by running the following command in your terminal:
++
+```
+rpk cloud login
+```
+
+. Follow the CLI output for next steps.
+
+== Manage your cluster
+
+You can interact with your Serverless cluster in Redpanda Cloud, similar to BYOC and Dedicated clusters. You can add team members and grant them access with ACLs on the **Security** page. 
+
+You can also interact with your Serverless cluster with `rpk` commands. For example:
+
+* xref:reference:rpk/rpk-cloud/rpk-cloud-login.adoc[`rpk cloud login`]: Use this to log in to Redpanda Cloud or to refresh the session.
+* xref:reference:rpk/rpk-topic.adoc[`rpk topic`]: Use this to manage topics, and produce and consume data. 
+* xref:reference:rpk/rpk-profile/rpk-profile-print.adoc[`rpk profile print`]: Use this to view `rpk` configuration and the URL for your Serverless cluster.
+* xref:reference:rpk/rpk-acl/rpk-acl-user.adoc[`rpk acl user`]: Use this to manage users and permissions. 
+
+== Run a demo application
+
+Redpanda provides code examples to create a topic and produce and consume messages. On the *Overview* page, in the *How to connect - Kafka API* tab, click *Python* or *Node.js* for code examples.
+
+For *Go*, run the following command in your terminal:
```

Comment:
Please lead with the `rpk generate app` example. This is the experience we want to lead with. More languages come later.